src/appmain.ts

```diff
@@ -741,18 +713,15 @@ export class ReplayWebApp extends LitElement {
                       IS_APP
                         ? html`
                             <p>
-                              ReplayWeb.page App is a standalone app for Mac,
-                              Windows and Linux that loads web archive files
+                              ReplayWeb.page is a standalone app for Mac,
+                              Windows, and Linux that loads archived items
                               provided by the user and renders them for replay.
                             </p>
                           `
                         : html` <p>
-                            <a href="https://replayweb.page" target="_blank"
-                              >ReplayWeb.page</a
-                            >
-                            is a browser-based viewer that loads web archive
-                            files provided by the user and renders them for
-                            replay in the browser.
+                            ReplayWeb.page is a browser-based viewer that loads
```

Comment:
Same as above, let's revert this change. The link is useful for opening a link to our site when using the embedded mode.